src/main/java/com/paloaltonetworks/osc/api/PANManagerSecurityGroupInterfaceApi.java

```diff
@@ -32,18 +32,18 @@
     static String apiKey = null;
     private VirtualSystemElement vs;
     private ApplianceManagerConnectorElement mc;
-    private ShowOperations showOperations;
+    private PanoramaApiClient client;
 
     public PANManagerSecurityGroupInterfaceApi(ApplianceManagerConnectorElement mc, VirtualSystemElement vs,
-            ShowOperations showOperations) {
+            PanoramaApiClient showOperations) {
         this.vs = vs;
         this.mc = mc;
-        this.showOperations = showOperations;
+        this.client = showOperations;
     }
 
     @Override
     public String createSecurityGroupInterface(SecurityGroupInterfaceElement sgiElement) throws Exception {
-        String status = this.showOperations.addDAGTag(sgiElement.getTag());
+        String status = this.client.addDAGTag(sgiElement.getTag());
```

Comment:
not sure i understand what this is doing.

We should iterate through all addresses belonging to the security group and add the  policy tags which are defined as org.osc.sdk.manager.element.SecurityGroupInterfaceElement.getManagerPolicyElements()

sgiElement.getTag() get encapsulation tag...which is not supported by PAN.